external/kafka-0-10/pom.xml

```diff
@@ -49,30 +50,14 @@
     </dependency>
     <dependency>
       <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>${kafka.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_${scala.binary.version}</artifactId>
-      <version>0.10.0.1</version>
-      <exclusions>
```

Comment:
Is the point here that it's not particularly important to carefully trim down the dependency graph if this becomes a test-only artifact?